### ISSUE TITLE
how can i train DORN (depth estimation method) on your KOD dataset?

### ISSUE THREAD

**Issue by maiminh1996** [WXinlong/ForeSeE#5]
Question is the same with the title.

**Comment by WXinlong** [comment]
@maiminh1996  You might need to organize the data as the DORN repo suggested.

**Comment by maiminh1996** [comment]
I found only 3622 for training set and 3696 for validation set annotations? Did you miss some annotations?
